### DISCUSSION TITLE
issues with installing ezkl in python

### DISCUSSION THREAD

**Discussion by chandang207** [zkonduit/ezkl#325] [Q&A] [answered]
When I run `import ezkl`, I get the error `ImportError: cannot import name 'forward' from 'ezkl_lib'`

<img width="630" alt="image" src="https://github.com/zkonduit/ezkl/assets/41006515/4aeb6299-2d45-4237-8eef-03b91832d0f7">

Can someone please help me

**Comment by alexander-camuto** [comment, marked as answer]
can you open this on pyezkl :) ? https://github.com/zkonduit/pyezkl
We're about to address this in https://github.com/zkonduit/pyezkl/pull/31
